src/client/test/components/matchResults/scoreKeepByInnings/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+
+import ScoreKeepByInnings from 'client/components/matchResults/scoreKeepByInnings';
+import { titles, buttonText } from 'client/constants/base';
+
+describe('<ScoreKeepByInnings />', () => {
+  afterEach(cleanup);
+
+  describe('render', () => {
+    it('should render correctly', () => {
+      const mockOnClick = jest.fn();
+      const { getByText, queryByText } = render(
+        <ScoreKeepByInnings isScoreKeepOpen onClick={mockOnClick} />
+      );
+
+      expect(queryByText(titles.scoreKeepByInning)).not.toBeNull();
+
+      fireEvent.click(getByText(buttonText.backToHome));
```

Comment:
Looks like this one is also for testing the click event. Maybe we should separate it from testing `render correctly`. 🤔

Reply:
I updated the test description to be more precise, please check, thanks :)